analysed-packages/pciutils/version-3.7.0/pciutils-3.7.0-OSS-disclosure.txt

```diff
@@ -0,0 +1,445 @@
+GPL-2.0-or-later, in this context, BSD-3-Clause has  been chosen. This choice
+shall not restrict the freedom of other users to select either BSD-3-Clause or
+GPL-2.0-or-later licenses. For convenience, all license texts are provided.
+
+--------------------------------------------------------------------------------
+
+Copyright notices
+
+copyright is held by Martin Mares and Albert Pool.
+Copyright (c) 2018 YuJian.Gong <gongyujian@acoinfo.com>
+Copyright (c) 2017 Joan Lledó <jlledom@member.fsf.org>
+Copyright (c) 2014 Martin Mares <mj@ucw.cz>
+Copyright (c) 2014 Gerd Hoffmann <kraxel@redhat.com>
+Copyright (c) 2013--2014 Tom Gundersen <teg@jklm.no>
+Copyright (c) 2013 Apple, Inc.
+Copyright (c) 2010, 2017 Rudolf Marek <r.marek@assembler.cz>
+Copyright (c) 2009 Francois Revol <revol@free.fr>
+Copyright (c) 2008--2009 Martin Mares <mj@ucw.cz>
+Copyright (c) 2008 Solarflare Communications
+Copyright (c) 2008 Martin Mares <mj@ucw.cz>
+Copyright (c) 2007--2008 Martin Mares <mj@ucw.cz>
+Copyright (c) 2007 Thomas Schwinge <tschwinge@gnu.org>
+Copyright (c) 2006 Samuel Thibault <samuel.thibault@ens-lyon.org> and Thomas
+Schwinge <tschwinge@gnu.org>
+Copyright (c) 2006 Martin Mares <mj@ucw.cz>
+Copyright (c) 2004 Alexander Stock <stock.alexander@gmx.de>
+Copyright (c) 2003--2006 Martin Mares <mj@ucw.cz>
+Copyright (c) 2003 Matthew Wilcox <matthew@wil.cx>
+Copyright (c) 2003 Martin Mares <mj@ucw.cz>
+Copyright (c) 2003 Marco Gerards <metgerards@student.han.nl>
+Copyright (c) 2003 Bill Moore <billm@eng.sun.com>
+Copyright (c) 2002 Quentin Garnier <cube@cubidou.net>
+Copyright (c) 2002 Martin Mares <mj@ucw.cz>
+Copyright (c) 1999 Jari Kirma <kirma@cs.hut.fi> Updated in 2003 by Samy Al Bahra
+<samy@kerneled.com> Updated in 2017 by Imre Vadász <imrevdsz@gmail.com>
+Copyright (c) 1999 Jari Kirma <kirma@cs.hut.fi>
+Copyright (c) 1998--2020 Martin Mares <mj@ucw.cz>
+Copyright (c) 1998--2014 Martin Mares <mj@ucw.cz>
+Copyright (c) 1997--2020 Martin Mares <mj@ucw.cz>
+Copyright (c) 1997--2018 Martin Mares <mj@ucw.cz>
+Copyright (c) 1997--2017 Martin Mares <mj@ucw.cz>
+Copyright (c) 1997--2016 Martin Mares <mj@ucw.cz>
+Copyright (c) 1997--2014 Martin Mares <mj@ucw.cz>
+Copyright (c) 1997--2013 Martin Mares <mj@ucw.cz>
+Copyright (c) 1997--2010 Martin Mares <mj@ucw.cz>
+Copyright (c) 1997--2008 Martin Mares <mj@ucw.cz>
+Copyright (c) 1997--2006 Martin Mares <mj@ucw.cz>
+Copyright (c) 1997--2003 Martin Mares <mj@ucw.cz>
+Copyright (c) 1997--2000 Martin Mares <mj@ucw.cz>
+Copyright (C) 1989, 1991 Free Software Foundation, Inc.,
+Copyright (C) 1989, 1990, 1991, 1992, 1993 Free Software Foundation, Inc.
+Copyright (C) 1987, 88, 89, 90, 91, 92, 1993 Free Software Foundation, Inc.
+(c) 2003--2012 Martin Mares <mj@ucw.cz>
+(c) 2003--2011 Martin Mares <mj@ucw.cz>
+(c) 1999--2014 Martin Mares <mj@ucw.cz>
+(c) 1998--2020 Martin Mares <mj@ucw.cz>
+(c) 1998--2013 Martin Mares <mj@ucw.cz>
+(C) MiniPCI 1610 ADSL AccessRunner
```

Comment:
Can you please remove the two entries:
(C) MiniPCI 1610 ADSL AccessRunner
(C) 8255 Digital Input / Output Card

These are no copyright notices. they are part of the pci.ids file